src/modules/users/index.ts

```diff
@@ -34,24 +62,16 @@ export class UsersModule extends BaseModule {
       email: string | null;
       oauth_provider: string | null;
       phone_number: string | null;
-    }>(`${this.sdk.apiBaseUrl}/v1/admin/auth/user/get`, this.sdk.secretApiKey, { issuer });
+      wallets: MagicWallet[] | null;
+    }>(`${this.sdk.apiBaseUrl}/v1/admin/auth/user/get?wallet_type=${walletType}`, this.sdk.secretApiKey, { issuer });
```

Comment:
What is the default wallet type? I am thinking of if this breaks existing integrations which assume ethereum.

Reply:
Default type is WalletType.None which will default to the existing behavior of `GET /v1/admin/auth/user/get`.